vagrant-base.json

```diff
@@ -114,8 +114,8 @@
     "iso_name": "ubuntu-16.04.1-server-amd64.iso",
     "memory": "1024",
     "metadata": "metadata.json",
-    "mirror": "http://releases.ubuntu.com",
-    "mirror_directory": "16.04",
+    "mirror": "http://old-releases.ubuntu.com/releases",
+    "mirror_directory": "16.04.1",
```

Comment:
Any plans for upgrading to 17 at some point or switching to a tumbleweed release of Debian or something? Just a thought so that updating this repo isn't as painful?